src/mappings/doid.sssom.tsv

```diff
@@ -0,0 +1,7 @@
+# curie_map:
+#   NCIT: http://purl.obolibrary.org/obo/NCIT_
+#   oboInOwl: http://www.geneontology.org/formats/oboInOwl#
+# license: http://w3id.org/sssom/license/unspecified
+# mapping_set_id: http://w3id.org/sssom/mappings/2485bf40-8891-47e4-b8ac-1f1ae0200f0c
+subject_id	predicate_id	object_id
```

Comment:
can you track down why in this case there is no "match_type" column?

Reply:
And also why this mapping set is empty - there are 1000 of mappings in doid.

Reply:
I ran the source file separately using `sssom-py` [`component-did.json`] and gives me an sssom file with mappings. I'm not sure what is happening within docker. I'll ig deeper

Reply:
Make sure that `make sssom` is run during the same time as you run sssom parse. The docker container does not retain installed packages between runs!